pkg/component/controller/kubeletconfig.go

```diff
@@ -236,22 +237,18 @@ func getDefaultProfile(dnsAddress string, clusterDomain string) unstructuredYaml
 	// - it's easier to merge programatically defined structure
 	// - apart from map[string]interface there is no good way to define free-form mapping
 
+	cipherSuites := make([]string, len(constant.AllowedTLS12CipherSuiteIDs))
+	for i, cipherSuite := range constant.AllowedTLS12CipherSuiteIDs {
+		cipherSuites[i] = tls.CipherSuiteName(cipherSuite)
+	}
+
```

Comment:
Maybe this could be a helper func in the `constant` package as it is used in couple different places now

Reply:
It's used in two places, one being deprecated. That's why didn't do it. Technically, the changes in this file aren't necessary at all, since it only exists for backwards compatibility when upgrading from k0s 1.25.